[2025-02-11 17:08:16,004: INFO: main: welcome to our custom log]
[2025-02-15 23:54:11,375: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-15 23:54:11,381: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-15 23:54:11,384: INFO: common: created directory at: artifacts]
[2025-02-15 23:54:11,386: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-15 23:54:11,388: INFO: 986331348: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-15 23:54:21,866: INFO: 986331348: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 00:32:37,389: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-16 00:32:37,390: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 00:32:37,394: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 00:32:37,394: INFO: common: created directory at: artifacts]
[2025-02-16 00:32:37,395: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-16 00:32:37,396: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 00:32:48,154: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 00:32:48,774: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-16 16:50:18,279: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 16:50:18,296: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 16:50:18,299: INFO: common: created directory at: artifacts]
[2025-02-16 16:51:15,416: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 16:51:15,421: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 16:51:15,422: INFO: common: created directory at: artifacts]
[2025-02-16 16:51:15,424: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-16 16:52:11,726: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 16:52:11,731: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 16:52:11,736: INFO: common: created directory at: artifacts]
[2025-02-16 16:52:11,737: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-16 16:52:19,702: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-02-16 17:02:35,851: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-16 17:02:35,855: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 17:02:35,858: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 17:02:35,860: INFO: common: created directory at: artifacts]
[2025-02-16 17:02:35,860: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "main.py", line 11, in <module>
    data_ingestion.main()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 14, in main
    data_ingestion_config = config.get_data_ingestion_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2025-02-16 17:11:49,411: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-16 17:11:49,413: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 17:11:49,416: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 17:11:49,417: INFO: common: created directory at: artifacts]
[2025-02-16 17:11:49,417: ERROR: main: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config']
Traceback (most recent call last):
  File "main.py", line 11, in <module>
    data_ingestion.main()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\pipeline\stage_01_data_ingestion.py", line 14, in main
    data_ingestion_config = config.get_data_ingestion_config()
AttributeError: 'ConfigurationManager' object has no attribute 'get_data_ingestion_config'
[2025-02-16 17:12:50,471: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-16 17:12:50,473: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 17:12:50,474: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 17:12:50,475: INFO: common: created directory at: artifacts]
[2025-02-16 17:12:50,475: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-16 17:12:50,475: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 17:13:01,542: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 17:13:02,254: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-16 17:13:02,254: INFO: main: *******************]
[2025-02-16 17:13:02,254: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-16 17:13:02,255: ERROR: main: name 'PrepareBaseModelTrainingPipeline' is not defined]
Traceback (most recent call last):
  File "main.py", line 21, in <module>
    prepare_base_model = PrepareBaseModelTrainingPipeline()
NameError: name 'PrepareBaseModelTrainingPipeline' is not defined
[2025-02-16 17:19:55,509: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-16 17:19:55,511: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 17:19:55,519: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 17:19:55,522: INFO: common: created directory at: artifacts]
[2025-02-16 17:19:55,523: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-16 17:19:55,523: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 17:20:07,236: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 17:20:07,853: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-16 17:20:07,854: INFO: main: *******************]
[2025-02-16 17:20:07,854: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-16 17:20:07,858: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 17:20:07,862: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 17:20:07,863: INFO: common: created directory at: artifacts]
[2025-02-16 17:20:07,864: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-16 17:20:08,776: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-02-16 17:20:09,538: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-02-16 22:05:25,751: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:05:25,761: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:05:25,764: INFO: common: created directory at: artifacts]
[2025-02-16 22:07:47,797: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:07:47,801: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:07:47,802: INFO: common: created directory at: artifacts]
[2025-02-16 22:07:47,805: INFO: common: created directory at: artifacts\training]
[2025-02-16 22:09:31,429: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:09:31,433: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:09:31,437: INFO: common: created directory at: artifacts]
[2025-02-16 22:09:31,439: INFO: common: created directory at: artifacts\training]
[2025-02-16 22:09:40,070: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:09:40,072: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:09:40,073: INFO: common: created directory at: artifacts]
[2025-02-16 22:09:40,075: INFO: common: created directory at: artifacts\training]
[2025-02-16 22:10:50,906: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:10:50,908: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:10:50,910: INFO: common: created directory at: artifacts]
[2025-02-16 22:10:50,914: INFO: common: created directory at: artifacts\training]
[2025-02-16 22:12:14,157: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:12:14,160: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:12:14,162: INFO: common: created directory at: artifacts]
[2025-02-16 22:12:14,164: INFO: common: created directory at: artifacts\training]
[2025-02-16 22:14:24,489: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:14:24,492: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:14:24,494: INFO: common: created directory at: artifacts]
[2025-02-16 22:14:24,495: INFO: common: created directory at: artifacts\training]
[2025-02-16 22:56:07,120: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-16 22:56:07,123: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:56:07,130: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:56:07,131: INFO: common: created directory at: artifacts]
[2025-02-16 22:56:07,132: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-16 22:56:07,132: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 22:56:19,472: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 22:56:20,368: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-16 22:56:20,369: INFO: main: *******************]
[2025-02-16 22:56:20,369: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-16 22:56:20,372: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:56:20,373: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:56:20,374: INFO: common: created directory at: artifacts]
[2025-02-16 22:56:20,374: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-16 22:56:21,120: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-02-16 22:56:21,831: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-02-16 22:56:21,831: INFO: main: *******************]
[2025-02-16 22:56:21,832: INFO: main: >>>>>> stage training started <<<<<<]
[2025-02-16 22:56:21,835: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 22:56:21,837: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 22:56:21,838: INFO: common: created directory at: artifacts]
[2025-02-16 22:56:21,838: ERROR: main: name 'os' is not defined]
Traceback (most recent call last):
  File "main.py", line 35, in <module>
    prepare_base_model.main()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\pipeline\stage_03_model_training.py", line 15, in main
    training_config = config.get_training_config()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\config\configuration.py", line 56, in get_training_config
    training_data = os.path.join(self.config.data_ingestion.unzip_dir, "kidney-ct-scan-image")
NameError: name 'os' is not defined
[2025-02-16 23:01:47,453: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-16 23:01:47,455: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 23:01:47,458: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 23:01:47,459: INFO: common: created directory at: artifacts]
[2025-02-16 23:01:47,460: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-16 23:01:47,460: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 23:01:57,048: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-16 23:01:57,913: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-16 23:01:57,915: INFO: main: *******************]
[2025-02-16 23:01:57,915: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-16 23:01:57,919: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 23:01:57,922: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 23:01:57,922: INFO: common: created directory at: artifacts]
[2025-02-16 23:01:57,923: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-16 23:01:58,794: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-02-16 23:01:59,661: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-02-16 23:01:59,661: INFO: main: *******************]
[2025-02-16 23:01:59,662: INFO: main: >>>>>> stage training started <<<<<<]
[2025-02-16 23:01:59,665: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-16 23:01:59,667: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-16 23:01:59,668: INFO: common: created directory at: artifacts]
[2025-02-16 23:01:59,668: INFO: common: created directory at: artifacts\training]
[2025-02-16 23:03:16,278: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2025-02-17 23:35:23,236: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 23:35:23,254: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 23:35:23,257: INFO: common: created directory at: artifacts]
[2025-02-17 23:35:48,423: INFO: common: json file saved at: scores.json]
[2025-02-17 23:39:50,125: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 23:39:50,142: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 23:39:50,148: INFO: common: created directory at: artifacts]
[2025-02-17 23:40:11,965: INFO: common: json file saved at: scores.json]
[2025-02-17 23:42:40,714: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 23:42:40,720: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 23:42:40,722: INFO: common: created directory at: artifacts]
[2025-02-17 23:43:04,121: INFO: common: json file saved at: scores.json]
[2025-02-17 23:46:54,188: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-17 23:46:54,196: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-17 23:46:54,198: INFO: common: created directory at: artifacts]
[2025-02-17 23:47:14,535: INFO: common: json file saved at: scores.json]
[2025-02-17 23:47:18,023: WARNING: save: Found untraced functions such as _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op, _jit_compiled_convolution_op while saving (showing 5 of 14). These functions will not be directly callable after loading.]
[2025-02-17 23:47:19,360: INFO: builder_impl: Assets written to: C:\Users\User\AppData\Local\Temp\tmpvb8xibn5\model\data\model\assets]
[2025-02-18 22:09:58,104: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-18 22:09:58,108: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:09:58,111: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:09:58,113: INFO: common: created directory at: artifacts]
[2025-02-18 22:09:58,113: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-18 22:09:58,114: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:10:09,960: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:10:10,630: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-18 22:10:10,630: INFO: main: *******************]
[2025-02-18 22:10:10,630: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-18 22:10:10,634: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:10:10,636: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:10:10,636: INFO: common: created directory at: artifacts]
[2025-02-18 22:10:10,637: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-18 22:10:10,637: ERROR: main: name 'PrepareBaseModel' is not defined]
Traceback (most recent call last):
  File "main.py", line 24, in <module>
    prepare_base_model.main()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 16, in main
    prepare_base_model = PrepareBaseModel(config=prepare_base_model_config)
NameError: name 'PrepareBaseModel' is not defined
[2025-02-18 22:23:36,879: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-18 22:23:36,882: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:23:36,885: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:23:36,887: INFO: common: created directory at: artifacts]
[2025-02-18 22:23:36,887: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-18 22:23:36,888: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:23:47,154: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:23:47,912: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-18 22:23:47,913: INFO: main: *******************]
[2025-02-18 22:23:47,913: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-18 22:23:47,915: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:23:47,917: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:23:47,917: INFO: common: created directory at: artifacts]
[2025-02-18 22:23:47,918: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-18 22:23:47,918: ERROR: main: name 'PrepareBaseModel' is not defined]
Traceback (most recent call last):
  File "main.py", line 24, in <module>
    prepare_base_model.main()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 16, in main
    prepare_base_model = PrepareBaseModel(config=prepare_base_model_config)
NameError: name 'PrepareBaseModel' is not defined
[2025-02-18 22:26:32,214: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-18 22:26:32,220: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:26:32,228: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:26:32,234: INFO: common: created directory at: artifacts]
[2025-02-18 22:26:32,235: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-18 22:26:32,237: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:26:43,986: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:26:45,382: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-18 22:26:45,382: INFO: main: *******************]
[2025-02-18 22:26:45,383: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-18 22:26:45,388: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:26:45,392: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:26:45,394: INFO: common: created directory at: artifacts]
[2025-02-18 22:26:45,396: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-18 22:26:45,400: ERROR: main: name 'PrepareBaseModel' is not defined]
Traceback (most recent call last):
  File "main.py", line 24, in <module>
    prepare_base_model.main()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 16, in main
    prepare_base_model = PrepareBaseModel(config=prepare_base_model_config)
NameError: name 'PrepareBaseModel' is not defined
[2025-02-18 22:28:35,983: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-18 22:28:35,986: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:28:35,988: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:28:35,989: INFO: common: created directory at: artifacts]
[2025-02-18 22:28:35,989: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-18 22:28:35,990: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:28:47,544: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:28:48,902: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-18 22:28:48,903: INFO: main: *******************]
[2025-02-18 22:28:48,903: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-18 22:28:48,912: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:28:48,920: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:28:48,922: INFO: common: created directory at: artifacts]
[2025-02-18 22:28:48,923: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-18 22:28:48,923: ERROR: main: name 'PrepareBaseModel' is not defined]
Traceback (most recent call last):
  File "main.py", line 24, in <module>
    prepare_base_model.main()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 16, in main
    prepare_base_model = PrepareBaseModel(config=prepare_base_model_config)
NameError: name 'PrepareBaseModel' is not defined
[2025-02-18 22:29:36,759: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-18 22:29:36,761: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:29:36,763: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:29:36,764: INFO: common: created directory at: artifacts]
[2025-02-18 22:29:36,765: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-18 22:29:36,765: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:29:46,672: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:29:47,359: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-18 22:29:47,360: INFO: main: *******************]
[2025-02-18 22:29:47,361: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-18 22:29:47,366: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:29:47,371: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:29:47,372: INFO: common: created directory at: artifacts]
[2025-02-18 22:29:47,373: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-18 22:29:47,374: ERROR: main: name 'PrepareBaseModel' is not defined]
Traceback (most recent call last):
  File "main.py", line 24, in <module>
    prepare_base_model.main()
  File "D:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\pipeline\stage_02_prepare_base_model.py", line 16, in main
    prepare_base_model = PrepareBaseModel(config=prepare_base_model_config)
NameError: name 'PrepareBaseModel' is not defined
[2025-02-18 22:30:37,502: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-18 22:30:37,504: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:30:37,506: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:30:37,506: INFO: common: created directory at: artifacts]
[2025-02-18 22:30:37,507: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-18 22:30:37,508: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:30:47,611: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:30:48,252: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-18 22:30:48,253: INFO: main: *******************]
[2025-02-18 22:30:48,258: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-18 22:30:48,261: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:30:48,265: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:30:48,266: INFO: common: created directory at: artifacts]
[2025-02-18 22:30:48,267: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-18 22:30:49,331: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-02-18 22:30:49,802: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-02-18 22:30:49,803: INFO: main: *******************]
[2025-02-18 22:30:49,803: INFO: main: >>>>>> stage training started <<<<<<]
[2025-02-18 22:30:49,807: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:30:49,809: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:30:49,811: INFO: common: created directory at: artifacts]
[2025-02-18 22:30:49,812: INFO: common: created directory at: artifacts\training]
[2025-02-18 22:31:58,180: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2025-02-18 22:31:58,180: INFO: main: *******************]
[2025-02-18 22:31:58,181: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2025-02-18 22:32:33,753: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-18 22:32:33,759: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:32:33,761: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:32:33,762: INFO: common: created directory at: artifacts]
[2025-02-18 22:32:33,762: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-18 22:32:33,763: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:32:43,978: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-18 22:32:44,639: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-18 22:32:44,640: INFO: main: *******************]
[2025-02-18 22:32:44,641: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-18 22:32:44,644: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:32:44,645: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:32:44,647: INFO: common: created directory at: artifacts]
[2025-02-18 22:32:44,647: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-18 22:32:45,260: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-02-18 22:32:45,634: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-02-18 22:32:45,634: INFO: main: *******************]
[2025-02-18 22:32:45,634: INFO: main: >>>>>> stage training started <<<<<<]
[2025-02-18 22:32:45,637: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-18 22:32:45,638: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-18 22:32:45,639: INFO: common: created directory at: artifacts]
[2025-02-18 22:32:45,640: INFO: common: created directory at: artifacts\training]
[2025-02-18 22:33:55,382: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2025-02-18 22:33:55,383: INFO: main: *******************]
[2025-02-18 22:33:55,383: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2025-02-19 14:23:36,009: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-19 14:23:36,021: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:23:36,034: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:23:36,046: INFO: common: created directory at: artifacts]
[2025-02-19 14:23:36,050: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-19 14:23:36,061: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-19 14:23:50,389: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-19 14:23:51,576: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-19 14:23:51,576: INFO: main: *******************]
[2025-02-19 14:23:51,576: INFO: main: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-19 14:23:51,578: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:23:51,580: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:23:51,581: INFO: common: created directory at: artifacts]
[2025-02-19 14:23:51,581: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-19 14:23:54,486: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-02-19 14:23:55,314: INFO: main: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-02-19 14:23:55,315: INFO: main: *******************]
[2025-02-19 14:23:55,316: INFO: main: >>>>>> stage training started <<<<<<]
[2025-02-19 14:23:55,321: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:23:55,324: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:23:55,325: INFO: common: created directory at: artifacts]
[2025-02-19 14:23:55,326: INFO: common: created directory at: artifacts\training]
[2025-02-19 14:25:09,091: INFO: main: >>>>>> stage training completed <<<<<<

x==========x]
[2025-02-19 14:25:09,092: INFO: main: *******************]
[2025-02-19 14:25:09,092: INFO: main: >>>>>> stage Evaluation stage started <<<<<<]
[2025-02-19 14:45:53,022: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-19 14:45:53,025: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:45:53,029: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:45:53,031: INFO: common: created directory at: artifacts]
[2025-02-19 14:45:53,032: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-19 14:45:53,032: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-19 14:46:03,509: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-19 14:46:04,066: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-19 14:46:10,393: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-02-19 14:46:10,396: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:46:10,398: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:46:10,399: INFO: common: created directory at: artifacts]
[2025-02-19 14:46:10,399: INFO: common: created directory at: artifacts/data_ingestion]
[2025-02-19 14:46:10,400: INFO: data_ingestion: Downloading data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-19 14:46:20,129: INFO: data_ingestion: Downloaded data from https://drive.google.com/file/d/1vlhZ5c7abUKF8xXERIw6m9Te8fW7ohw3/view?usp=sharing into file artifacts/data_ingestion/data.zip]
[2025-02-19 14:46:20,803: INFO: stage_01_data_ingestion: >>>>>> stage Data Ingestion stage completed <<<<<<

x==========x]
[2025-02-19 14:46:34,518: INFO: stage_02_prepare_base_model: *******************]
[2025-02-19 14:46:34,518: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model started <<<<<<]
[2025-02-19 14:46:34,522: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:46:34,526: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:46:34,528: INFO: common: created directory at: artifacts]
[2025-02-19 14:46:34,528: INFO: common: created directory at: artifacts/prepare_base_model]
[2025-02-19 14:46:35,239: WARNING: saving_utils: Compiled the loaded model, but the compiled metrics have yet to be built. `model.compile_metrics` will be empty until you train or evaluate the model.]
[2025-02-19 14:46:35,751: INFO: stage_02_prepare_base_model: >>>>>> stage Prepare base model completed <<<<<<

x==========x]
[2025-02-19 14:46:42,052: INFO: stage_03_model_training: *******************]
[2025-02-19 14:46:42,053: INFO: stage_03_model_training: >>>>>> stage training started <<<<<<]
[2025-02-19 14:46:42,053: ERROR: stage_03_model_training: name 'ModelTrainingPipeli' is not defined]
Traceback (most recent call last):
  File "src/cnnClassifier/pipeline/stage_03_model_training.py", line 27, in <module>
    obj = ModelTrainingPipeli()
NameError: name 'ModelTrainingPipeli' is not defined
[2025-02-19 14:47:32,567: INFO: stage_03_model_training: *******************]
[2025-02-19 14:47:32,568: INFO: stage_03_model_training: >>>>>> stage training started <<<<<<]
[2025-02-19 14:47:32,570: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:47:32,571: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:47:32,572: INFO: common: created directory at: artifacts]
[2025-02-19 14:47:32,573: INFO: common: created directory at: artifacts\training]
[2025-02-19 14:48:41,149: INFO: stage_03_model_training: >>>>>> stage training completed <<<<<<

x==========x]
[2025-02-19 14:51:45,238: INFO: stage_04_model_evaluation: *******************]
[2025-02-19 14:51:45,238: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2025-02-19 14:51:45,240: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:51:45,242: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:51:45,242: INFO: common: created directory at: artifacts]
[2025-02-19 14:51:45,242: ERROR: stage_04_model_evaluation: name 'Evaluation' is not defined]
Traceback (most recent call last):
  File "src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 27, in <module>
    obj.main()
  File "src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 16, in main
    evaluation = Evaluation(eval_config)
NameError: name 'Evaluation' is not defined
[2025-02-19 14:52:28,356: INFO: stage_04_model_evaluation: *******************]
[2025-02-19 14:52:28,357: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2025-02-19 14:52:28,359: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:52:28,361: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:52:28,361: INFO: common: created directory at: artifacts]
[2025-02-19 14:52:28,808: ERROR: stage_04_model_evaluation: name 'model' is not defined]
Traceback (most recent call last):
  File "src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 27, in <module>
    obj.main()
  File "src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 17, in main
    evaluation.evaluation()
  File "d:\data science projects\kedneydiseaseclasifficationdeeplearning\src\cnnClassifier\components\model_evaluation_mlflow.py", line 48, in evaluation
    self.score = model.evaluate(self.valid_generator)
NameError: name 'model' is not defined
[2025-02-19 14:58:25,470: INFO: stage_04_model_evaluation: *******************]
[2025-02-19 14:58:25,471: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2025-02-19 14:58:25,473: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:58:25,476: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:58:25,477: INFO: common: created directory at: artifacts]
[2025-02-19 14:58:25,478: ERROR: stage_04_model_evaluation: name 'Evaluation' is not defined]
Traceback (most recent call last):
  File "src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 27, in <module>
    obj.main()
  File "src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 16, in main
    evaluation = Evaluation(eval_config)
NameError: name 'Evaluation' is not defined
[2025-02-19 14:59:34,854: INFO: stage_04_model_evaluation: *******************]
[2025-02-19 14:59:34,854: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2025-02-19 14:59:34,856: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 14:59:34,858: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 14:59:34,859: INFO: common: created directory at: artifacts]
[2025-02-19 14:59:34,860: ERROR: stage_04_model_evaluation: __init__() takes 1 positional argument but 2 were given]
Traceback (most recent call last):
  File "src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 27, in <module>
    obj.main()
  File "src/cnnClassifier/pipeline/stage_04_model_evaluation.py", line 16, in main
    evaluation = Evaluation(eval_config)
TypeError: __init__() takes 1 positional argument but 2 were given
[2025-02-19 15:01:10,783: INFO: stage_04_model_evaluation: *******************]
[2025-02-19 15:01:10,783: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage started <<<<<<]
[2025-02-19 15:01:10,785: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-02-19 15:01:10,787: INFO: common: yaml file: params.yaml loaded successfully]
[2025-02-19 15:01:10,788: INFO: common: created directory at: artifacts]
[2025-02-19 15:01:29,894: INFO: common: json file saved at: scores.json]
[2025-02-19 15:01:29,895: INFO: stage_04_model_evaluation: >>>>>> stage Evaluation stage completed <<<<<<

x==========x]
